assets/src/edit-story/components/panels/stylePreset/utils.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import deepEqual from 'deep-equal';
+
+export function findMatchingColor(color, stylePresets, isText) {
+  // Ensure that the color doesn't already exist.
```

Comment:
This comment doesn't really say anything relevant to this function?

Reply:
Ah, yes, that was a left-over from moving the logic to util, removing.